build.py

```diff
@@ -64,21 +64,28 @@
 # different versions are used then one backend or the other will
 # incorrectly load the other version of the openvino libraries.
 #
+# Adding another openvino version. Take note of the above issue
+# when loading mulitple backends with different openvino versions.
+# To use openvino_21_4, the model configuration should specify
+# the "openvino_21_4" in the "backend" field.
+#
 TRITON_VERSION_MAP = {
     '2.18.0dev': (
         '22.01dev',  # triton container
         '21.11',  # upstream container
         '1.10.0', # ORT
         '2021.2.200',  # ORT OpenVINO
         '2021.2',  # Standalone OpenVINO
-        '2.2.9')  # DCGM version
+        '2.2.9', # DCGM version
+        '2021.4.582') # Standalone OpenVINO for openvino_21_4
 }
 
 EXAMPLE_BACKENDS = ['identity', 'square', 'repeat']
 CORE_BACKENDS = ['ensemble']
 NONCORE_BACKENDS = [
     'tensorflow1', 'tensorflow2', 'onnxruntime', 'python', 'dali', 'pytorch',
-    'openvino', 'fil', 'fastertransformer', 'tensorrt', 'armnn_tflite'
+    'openvino', 'fil', 'fastertransformer', 'tensorrt', 'armnn_tflite',
+    'openvino_21_4'
```

Comment:
From the list above generate 'openvino_2021_2' and 'openvino_2021_4' backends here instead of hard-coding them